src/graphql/root/mutation/ln-invoice-fee-probe.ts

```diff
@@ -34,13 +36,28 @@ const LnInvoiceFeeProbeMutation = GT.Field({
     const btcWalletValidated = await validateIsBtcWalletForMutation(walletId)
     if (btcWalletValidated != true) return btcWalletValidated
 
-    const feeSatAmount = await Payments.getLightningFeeEstimation({
+    const { result: feeSatAmount, error } = await Payments.getLightningFeeEstimation({
       walletId,
       paymentRequest,
     })
-    if (feeSatAmount instanceof Error) {
-      const appErr = mapError(feeSatAmount)
-      return { errors: [{ message: appErr.message }] }
+
+    if (feeSatAmount && error instanceof Error) {
```

Comment:
this should be `if (feeSatAmount >= 0` because 0 is a valid value for `feeSatAmount`

Reply:
Done: https://github.com/GaloyMoney/galoy/pull/1343/commits/2ad1941090bea25dc152e43c21d1b1a887123c05